integration_tests/setup_test.go

```diff
@@ -60,7 +61,7 @@ func MNEMONICS() []string {
 var (
 	stakeAmount, _  = sdk.NewIntFromString("100000000000")
 	stakeAmountCoin = sdk.NewCoin(bondDenom, stakeAmount)
-	hardhatCellar   = common.HexToAddress("0x4C4a2f8c81640e47606d3fd77B353E87Ba015584")
+	hardhatCellar   = common.HexToAddress("0x0000000000000000000000000000000000000000")
```

Comment:
It could just be a defaulted string. `hardhatCellar` gets dynamically set by parsing the logs for whatever the address is now just like the gravity contract.

Reply:
Mmm, I see that happens for gravity contract too. Might be worth a comment.